spec/controllers/taxa_controller_spec.rb

```diff
@@ -314,7 +314,8 @@
             "0" => {
               "id" => cs.id,
               "status" => cs.status,
-              "authority" => cs.authority
```

Comment:
Without geoprivacy it was changing from default (obscured), to ""/nil (open). Might be missing something with the way the params are parsed, but I think this makes sense?